functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/pages/jobs/JobCreatePage.groovy

```diff
@@ -0,0 +1,487 @@
+    By saveOptionBy = By.xpath("//*[@title[contains(.,'Save the new option')]]")
+    By nodeDispatchTrueBy = By.id("doNodedispatchTrue")
+    By nodeFilterLinkBy = By.cssSelector("#job_edit__node_filter_include .job_edit__node_filter__filter_select_dropdown")
+    By nodeFilterSelectAllLinkBy = By.cssSelector("#job_edit__node_filter_include .job_edit__node_filter__filter_select_all")
+    By nodeMatchedCountBy = By.xpath("//span[@class='text-info node_filter_results__matched_nodes_count']")
+    By excludeFilterTrueBy = By.xpath("//*[@id='excludeFilterTrue']")
+    By editableFalseBy = By.xpath("//*[@id='editableFalse']")
+    By schedJobNodeThreadCountBy = By.xpath("//*[@id='schedJobnodeThreadcount']")
+    By schedJobNodeRankAttributeBy = By.xpath("//*[@id='schedJobnodeRankAttribute']")
+    By nodeRankOrderDescendingBy = By.xpath("//*[@id='nodeRankOrderDescending']")
+    By nodeKeepGoingTrueBy = By.xpath("//*[@id='nodeKeepgoingTrue']")
+    By successOnEmptyNodeFilterTrueBy = By.xpath("//*[@id='successOnEmptyNodeFilterTrue']")
+    By nodesSelectedByDefaultFalseBy = By.xpath("//*[@id='nodesSelectedByDefaultFalse']")
+    By orchestratorDropdownBy = By.cssSelector('#orchestrator-edit-type-dropdown > button')
+    By errorAlertBy = By.cssSelector('#error')
+    By formValidationAlertBy = By.cssSelector('#page_job_edit > div.list-group-item > div.alert.alert-danger')
+    By sessionSectionBy = By.xpath("//div[contains(@class, 'opt_sec_nexp_disabled')]")
+    By secureInputTypeBy = By.xpath("//input[contains(@value, 'secureExposed')]")
+    By optionOpenKeyStorageBy = By.cssSelector(".btn.btn-default.obs-select-storage-path")
+    By optionCloseKeyStorageBy = By.xpath("//button[@class='btn btn-sm btn-default']")
+    By optionUndoBy = By.xpath("//*[@id='optundoredo']/div/span[1]")
+    By optionRedoBy = By.xpath("//*[@id='optundoredo']/div/span[2]")
+    By optionRevertAllBy = By.xpath("//*[starts-with(@id,'revertall')]")
+    By optionConfirmRevertAllBy = By.cssSelector("div[class='popover-content'] span[class*='confirm']")
+    By workFlowStepBy = By.linkText("Workflow Steps")
+    By ansibleBinariesPathBy = By.name("pluginConfig.ansible-binaries-dir-path")
+    By autocompleteSuggestionsBy = By.cssSelector("div[class='autocomplete-suggestions']")
+    By wfUndoButtonBy = By.xpath("//*[@id='wfundoredo']/div/span[1]")
+    By wfRedoButtonBy = By.xpath("//*[@id='wfundoredo']/div/span[2]")
+    By wfRevertAllButtonBy = By.xpath("//*[@id='wfundoredo']/div/span[3]")
+    By revertWfConfirmBy = By.xpath('//*[starts-with(@id,"popover")]/div[2]/span[2]')
+    By listWorkFlowItemBy = By.xpath("//*[starts-with(@id,'wfitem_')]")
+    By addSimpleCommandStepBy = By.xpath("//span[contains(@onclick, 'wfnewbutton')]")
+
+    String loadPath = "/job/create"
+
+    JobCreatePage(final SeleniumContext context, String project) {
+        super(context)
+        if (project.split('##').length > 1) {
```

Comment:
what is this block for?

Reply:
Removed, but it was so that the constructor redirects directly to the job id. And when the split is identified, go directly to the job.